apps/src/code-studio/pd/application_dashboard/cohort_view.jsx

```diff
@@ -126,6 +130,12 @@ class CohortView extends React.Component {
             >
               Download CSV
             </Button>
+            <Button
+              style={styles.button}
+              onClick={this.handleViewAllClick}
+            >
+              View All Applications
```

Comment:
Not sure if this is the capitalization we want - I think it should be lowercase `all applications`